Weather/Models/DailyWeatherData.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DailyWeatherData.swift
+//  Weather
+//
+//  Created by Alex on 31.07.22.
+//
+
+import Foundation
+
+struct DailyWeatherData: Codable, Identifiable {
+    var dt: Int
+    var temp: TemperatureData
+    var weather: [WeatherDetail]
+    
+    enum CodingKey: String {
+        case dt
+        case temp
+        case weather
```

Comment:
There was no need to create `CodingKey` enum in you case because your properties names are equal to the API properties names. BTW, you created your enum incorrect:
```
enum CodingKeys: String, CodingKey {
```

Reply:
Fixed.)